contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,286 @@
+---
+date: 2024-11-13
+title: How we found our activation metric (and how you can too)
+author:
+  - raquel-m-smith
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/hog_ql.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Product analytics
+  - Product engineers
+  - Growth
+  - Inside PostHog
+crosspost:
+  - Blog
+---
+
+People are signing up for your product. Great! But before you start planning your IPO, you need to make sure they actually stick around. This is the goal of **activation**.
+
+The trouble with activation is that it is hard to define. Many say it is when a user reaches the "aha" moment of your product, but this is vague and different for everyone.
+
+For example, we have 8+ products. Each product has 10+ events that **could** be part of its activation metric. This means we spent a lot of time discovering our products' activation metrics. We're sharing our process here to help you do the same.
+
+But before we jump in... what is activation _really_?
+
+## What is an activation metric?
+
+Activation metrics are a set of actions a new user performs that correlate with greater retention.
+
+Focusing on retention solves the trickiest part of defining an activation metric: it could be _anything_. Does completing onboarding count as activation? Creating their first object? How about inviting a teammate? Depending on your product, any or all of these could be an activation metric. It is **retention** that makes it clear.
```

Comment:
Since this paragraph leads with "focusing on retention", it reads better to focus the rest of the paragraph on it to highlight what we mean. Else its a bit confusing why we mean you should focus on retention
```suggestion
Focusing on retention solves the trickiest part of defining an activation metric: Does completing onboarding mean a user is going to retain? What about creating their first object? Inviting a teammate? Depending on your product, any or all of these could correlate with increased retention, and thus could be your activation metric
```